app/imports/startup/server/publications.jsx

```diff
@@ -0,0 +1,12 @@
+import { Meteor } from 'meteor/meteor';
+
+Meteor.publish('users.workers', function() {
+
+  if (Roles.userIsInRole(this.userId, 'admin')) {
```

Comment:
this raises the question of if we should lock down all subscriptions to users/roles. Probably so and therefor the projects list subscription needs that added